UM/TaskManagement/HttpRequestScope.py

```diff
@@ -6,20 +6,25 @@
 from UM.Application import Application
 
 
-## Modifies a request in some way. This concept is sometimes called a request interceptor. Could be used to add
-# authorization headers or set user agents, for example
 class HttpRequestScope:
-    ## Invoked after request-specific headers are set and before HttpRequestData is created
+    """Modifies a request in some way. This concept is sometimes called a request interceptor.
+
+    Could be used to add authorization headers or set user agents, for example
+    """
+
     def request_hook(self, request: QNetworkRequest):
```

Comment:
Hmm, this should have been requestHook (since it's a function).

Reply:
Done